packages/zosfiles/__tests__/__system__/api/methods/hMigrate/HMigrate.system.test.ts

```diff
@@ -0,0 +1,141 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Create, Delete, CreateDataSetTypeEnum, HMigrate, ZosFilesMessages } from "../../../../..";
+import { Imperative, Session } from "@zowe/imperative";
+import { inspect } from "util";
+import { ITestEnvironment } from "../../../../../../../__tests__/__src__/environment/doc/response/ITestEnvironment";
+import { TestEnvironment } from "../../../../../../../__tests__/__src__/environment/TestEnvironment";
+import { ITestPropertiesSchema } from "../../../../../../../__tests__/__src__/properties/ITestPropertiesSchema";
+import { List, IListOptions } from "../../../../../src/api";
+import { IMigrateOptions } from "../../../../../src/api/methods/hMigrate/doc/IMigrateOptions";
+
+let REAL_SESSION: Session;
+let testEnvironment: ITestEnvironment;
+let defaultSystem: ITestPropertiesSchema;
+let dataSet1: string;
+let dataSet2: string;
+
+const listOptions: IListOptions = { attributes: true };
+
+describe("Migrate Dataset", () => {
+    beforeAll(async () => {
+        testEnvironment = await TestEnvironment.setUp({ testName: "zos_file_migrate" });
+        defaultSystem = testEnvironment.systemTestProperties;
+
+        REAL_SESSION = TestEnvironment.createZosmfSession(testEnvironment);
+        dataSet1 = `${defaultSystem.zosmf.user.trim().toUpperCase()}.SDATA.SET`;
+        dataSet2 = `${defaultSystem.zosmf.user.trim().toUpperCase()}.PDATA.SET`;
+    });
+
+    afterAll(async () => {
+        await TestEnvironment.cleanUp(testEnvironment);
+    });
+
+    beforeEach(async () => {
+        try {
+            await Create.dataSet(REAL_SESSION, CreateDataSetTypeEnum.DATA_SET_SEQUENTIAL, dataSet1);
+            await Create.dataSet(REAL_SESSION, CreateDataSetTypeEnum.DATA_SET_PARTITIONED, dataSet2);
+        } catch (err) {
+            Imperative.console.info(`Error: ${inspect(err)}`);
+        }
+    });
+    afterEach(async () => {
+        try {
+            await Delete.dataSet(REAL_SESSION, dataSet1);
+            await Delete.dataSet(REAL_SESSION, dataSet2);
+        } catch (err) {
+            Imperative.console.info(`Error: ${inspect(err)}`);
+        }
+    });
+    describe("Success Scenarios", () => {
+        it("Migrate sequential", async () => {
```

Comment:
`it` blocks normally contain a sentence starting with `should` (e.g. `it("should migrate a sequential dataset")`)

Reply:
Hey @tucker01, thanks for your comments, I have changed the `it` blocks so that they start with a `should`.